tests/docker_cluster.py

```diff
@@ -290,7 +290,7 @@ def _are_centos_container_services_up(self, host):
             return False
         return True
 
-    def exec_cmd_on_container(self, host, cmd, raise_error=True, tty=False):
+    def exec_cmd(self, host, cmd, raise_error=True, tty=False):
```

Comment:
Could you call this exec_cmd_on_host?  Then it will match write_content_to_host.  though the argument order is different.